pkgs/data/icons/catppuccin-papirus-folders/default.nix

```
@@ -0,0 +1,56 @@
+{ stdenvNoCC
+, lib
+, fetchFromGitHub
+, papirus-icon-theme
+, gtk3
+, bash
+, getent
+, flavor ? "mocha"
+, accent ? "blue"
+, ...
+}:
+let
+  validAccents = [ "blue" "flamingo" "green" "lavender" "maroon" "mauve" "peach" "pink" "red" "rosewater" "sapphire" "sky" "teal" "yellow" ];
+  validFlavors = [ "latte" "frappe" "macchiato" "mocha" ];
+  pname = "catppuccin-papirus-folders";
+in
+lib.checkListOfEnum "${pname}: accent color" validAccents [ accent ]
+  lib.checkListOfEnum "${pname}: flavor"
+  validFlavors [ flavor ]
+  stdenvNoCC.mkDerivation
+rec {
+  inherit pname;
+  version = "unstable-2022-12-04";
+
+  src = fetchFromGitHub {
+    owner = "catppuccin";
+    repo = "papirus-folders";
+    rev = "1a367642df9cf340770bd7097fbe85b9cea65bcb";
+    sha256 = "sha256-mFDfRVDA9WyriyFVzsI7iqmPopN56z54FvLkZDS2Dv8=";
+  };
+
+  buildInputs = [ bash getent gtk3 papirus-icon-theme ];
+
+  patchPhase = ''
+    substituteInPlace ./papirus-folders --replace "getent" "${getent}/bin/getent"
+  '';
```

```suggestion
  postPatch = ''
    patchShebangs ./papirus-folders
  '';
```

You only need to `patchShebangs` to get this script to run without bash and if you set `USER_HOME` and `DISABLE_UPDATE_ICON_CACHE`, it doesn't need `getent`

Also `postPatch` for that sort of thing is better than `patchPhase` (in case someone overrides with a particular patch file)